### PR TITLE
[Ubuntu] Fix kubectl version report

### PR DESCRIPTION
**Pull request by ddobranic**
# Description
Fix for `kubectl` version in software report.

#### Related issue:

https://github.com/actions/virtual-environments-internal/issues/3680#issuecomment-1117011303

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 1]
```diff
@@ -142,7 +142,7 @@ function Get-KindVersion {
 }
 
 function Get-KubectlVersion {
-    $kubectlVersion = kubectl version --client --short | Take-OutputPart -Part 2 | Take-OutputPart -Part 0 -Delimiter "v"
+    $kubectlVersion = kubectl version --client --short | Select-Object -Last 2 | Select-Object -First 1 | Take-OutputPart -Part 2 | Take-OutputPart -Part 0 -Delimiter "v"
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:145]: You should parse version from json: `kubectl version --client --output=json`

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 2]
```diff
@@ -142,7 +142,7 @@ function Get-KindVersion {
 }
 
 function Get-KubectlVersion {
-    $kubectlVersion = kubectl version --client --short | Take-OutputPart -Part 2 | Take-OutputPart -Part 0 -Delimiter "v"
+    $kubectlVersion = kubectl version --client --output=json | ConvertFrom-Json | Select-Object -ExpandProperty clientVersion | Select-Object -ExpandProperty gitVersion | Take-OutputPart -Part 0 -Delimiter "v"
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:145]: nitpick: `(kubectl version --client --output=json | ConvertFrom-Json).clientVersion.gitVersion.Replace('v','')`

Comment (ddobranic) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:145]: Looks better. Thanks.

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 2 above)
Comment (miketimofeev) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1]: I think we can do it even easier
```pwsh
(kubectl version --client --output=json | ConvertFrom-Json).clientVersion.gitVersion.trim("v")
```

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**